graylog2-server/src/main/java/org/graylog2/events/ClusterEventPeriodical.java

```diff
@@ -86,16 +86,18 @@ static DBCollection prepareCollection(final MongoConnection mongoConnection) {
 
         DBCollection coll = db.getCollection(COLLECTION_NAME);
 
+        /* FIXME: Not supported by Fongo
         if (coll.isCapped()) {
             LOG.warn("The \"{}\" collection in MongoDB is capped which will cause problems. Please drop the collection.", COLLECTION_NAME);
         }
+        */
 
         coll.createIndex(DBSort
                 .asc("timestamp")
                 .asc("producer")
                 .asc("consumers"));
 
-        coll.setWriteConcern(WriteConcern.FSYNCED);
+        coll.setWriteConcern(WriteConcern.JOURNALED);
```

Comment:
We are using `FSYNCED` in some other places as well. Should we switch those as well? 
```
$ git grep -F WriteConcern.
graylog2-server/src/main/java/org/graylog2/cluster/ClusterConfigServiceImpl.java:        coll.setWriteConcern(WriteConcern.FSYNCED);
graylog2-server/src/main/java/org/graylog2/cluster/ClusterConfigServiceImpl.java:        dbCollection.update(DBQuery.is("type", canonicalClassName), clusterConfig, true, false, WriteConcern.FSYNCED);
graylog2-server/src/main/java/org/graylog2/database/MongoConnectionImpl.java:            db.setWriteConcern(WriteConcern.SAFE);
graylog2-server/src/main/java/org/graylog2/events/ClusterEventCleanupPeriodical.java:            final WriteResult<ClusterEvent, String> writeResult = dbCollection.remove(query, WriteConcern.FSYNCED);
graylog2-server/src/main/java/org/graylog2/events/ClusterEventPeriodical.java:        coll.setWriteConcern(WriteConcern.JOURNALED);
graylog2-server/src/main/java/org/graylog2/events/ClusterEventPeriodical.java:            final String id = dbCollection.save(clusterEvent, WriteConcern.JOURNALED).getSavedId();
graylog2-server/src/main/java/org/graylog2/metrics/MongoDbMetricsReporter.java:            collection.insert(docs, WriteConcern.UNACKNOWLEDGED);
graylog2-server/src/test/java/org/graylog2/cluster/ClusterConfigServiceImplTest.java:        assertThat(collection.getWriteConcern()).isEqualTo(WriteConcern.FSYNCED);
graylog2-server/src/test/java/org/graylog2/cluster/ClusterConfigServiceImplTest.java:        assertThat(collection.getWriteConcern()).isEqualTo(WriteConcern.FSYNCED);
graylog2-server/src/test/java/org/graylog2/events/ClusterEventPeriodicalTest.java:        assertThat(collection.getWriteConcern()).isEqualTo(WriteConcern.JOURNALED);
graylog2-server/src/test/java/org/graylog2/events/ClusterEventPeriodicalTest.java:        assertThat(collection.getWriteConcern()).isEqualTo(WriteConcern.JOURNALED);
```